spec/queries/find_fixity_events_spec.rb

```diff
@@ -60,6 +60,25 @@
       end
     end
 
+    context "when a fixity check which has failed later succeeds" do
+      # Old one uses preservation objects and real resources, this one doesn't; not sure if necessary
+      let(:event) { FactoryBot.create_for_repository(:cloud_fixity_event, status:  "FAILURE", resource_id: event_resource_id, current: true) }
+      let(:event2) { FactoryBot.create_for_repository(:cloud_fixity_event, status: "FAILURE", resource_id: event2_resource_id) }
+      let(:event3_resource_id) { Valkyrie::ID.new(SecureRandom.uuid) }
+      let(:event3) { FactoryBot.create_for_repository(:cloud_fixity_event, status: "SUCCESS", resource_id: event3_resource_id, current: true) }
```

Comment:
It looks like for this test we want 2 events with the same resource id, where the first failed and the second succeeded. and then we want a 3rd event with a different resource id and a failed status. the query should return only the 3rd event, since the resource connected to the first 2 events has now had a successful fixity check.

Reply:
Oops, didn't notice that the first two events used the same id. Just changed it!